docs/InstallGuide.md

```diff
@@ -131,294 +138,221 @@ To automatically launch the mysql server, execute the following command.
 
-Please install OpenVNet according to the "Pre-setup" and "OpenVNet installation" sections of "Preliminary Operations and Installation".
+#### Datapath
 
-The config files (vnmgr.conf, vna.conf and webapi.conf) in the /etc/openvnet directory do not need to be edited. They're set up for a single node installation by default.
+```
+# ./bin/vnctl datapaths add --uuid dp-test1 --display-name test1 --dpid 0x0000aaaaaaaaaaaa --node-id vna
+```
 
-### Other package installation
+* dpid
 
-    # yum install qemu-kvm
-    # yum install tcpdump
+The datapath ID specified in `/etc/sysconfig/network-scripts/ifcfg-br0`
 
+* node-id
 
-Start the required processes for OpenVNet
------------------------------------------
+The ID of the vna written in `/etc/openvnet/vna.conf`
 
-### redis
 
-    # chkconfig --list redis
-    redis           0:off   1:off   2:off   3:off   4:off   5:off   6:off
-    # chkconfig redis on
-    # chkconfig --list redis
-    redis           0:off   1:off   2:on    3:on    4:on    5:on    6:off
-    # service redis start
+#### Network
 
+```
+# ./bin/vnctl networks add --uuid nw-test1 --display-name testnet1 --ipv4-network 10.100.0.0 --ipv4-prefix 24 --network-mode virtual
+```
 
-### mysqld
+* ipv4-network
 
-    # chkconfig --list mysqld
-    mysqld          0:off   1:off   2:off   3:off   4:off   5:off   6:off
-    # chkconfig mysqld on
-    mysqld          0:off   1:off   2:on    3:on    4:on    5:on    6:off
-    # service mysqld start
+The IPv4 network address.
 
+* ipv4-prefix
 
-Start OpenVNet
----------------
-    # reboot
-    #
-    # initctl start vnet-vnmgr
-    # initctl start vnet-webapi
-    # initctl start vnet-vna
-    # initctl list | grep vnet # <- This line confirms that all three processes have started successfully.
-    vnet-vna start/running, process 2505
-    vnet-vnmgr start/running, process 2286
-    vnet-webapi start/running, process 2296
+The IPv4 network prefix. (default 24)
 
+* network-mode
 
-Download test environment
---------------------------
+The mode of the network to create. Use `virtual` in case of creating virtual network.
 
-Download the test environment archive and unpack it in the /root directory. The size of this archive is about 1.3 GB.
+#### Interface
 
-     # curl -o /root/vnet-test-kvm.tgz -R http://dlc.openvnet.axsh.jp/tests/vnet-test-kvm.tgz
-     # tar xvfz ./vnet-test-kvm.tgz
+```
+# ./bin/vnctl interfaces add --uuid if-inst1 --mode vif --owner-datapath-uuid dp-test1 --mac-address 10:54:ff:00:00:01 --network-uuid nw-test1 --ipv4-address 10.100.0.10 --port-name inst1
+# ./bin/vnctl interfaces add --uuid if-inst2 --mode vif --owner-datapath-uuid dp-test1 --mac-address 10:54:ff:00:00:02 --network-uuid nw-test1 --ipv4-address 10.100.0.11 --port-name inst2
+```
 
-Create Database
-----------------
+* mode
 
-Create the database and enter the sample data.
+The mode of the interface. `vif` must be specified if a virtual machine is supposed to have this.
```

Comment:
Is this really the reason to specify `vif`? I think OpenVNet doesn't care that it's a virtual machine attached to this interface.

Looking at the [list of interface modes](https://github.com/axsh/openvnet/blob/master/vnet/lib/vnet/constants/interface.rb), I believe `vif` means that it's going to be a network interface directly connected to a virtual network.